coreupdate/configure-machines.md

```diff
@@ -52,6 +66,20 @@ coreos:
 
 More information can be found in the [cloud-config guide](http://coreos.com/docs/cluster-management/setup/cloudinit-cloud-config/#coreos).
 
+Or the Ignition config:
+
+```json
+{
+  "ignition": { "version": "2.0.0" },
+  "files": [{
+    "filesystem": "root",
+    "path": "/etc/coreos/update.conf",
+    "mode": 420,
```

Comment:
Decimal modes look WEIRD. This is octal 644, rw-r--r-- (not a correction, just a note for my own ref)